docs/codeql/codeql-language-guides/functions-in-cpp.rst

```diff
@@ -58,11 +56,9 @@ You can modify the query to remove functions where a function pointer is used to
      and not exists(FunctionAccess fa | fa.getTarget() = f)
    select f, "This function is never called, or referenced with a function pointer."
 
-➤ `See this in the query console on LGTM.com <https://lgtm.com/query/1505890446605/>`__
-
 This query returns fewer results. However, if you examine the results then you can probably still find potential refinements.
 
-For example, there is a more complicated LGTM `query <https://lgtm.com/rules/2152580467/>`__ that finds unused static functions. To see the code for this query, click **Open in query console** at the top of the page.
+For example, there is a more complicated standard query, `Unused static function <https://codeql.github.com/codeql-query-help/cpp/cpp-unused-static-function/>`__, that finds unused static functions.
 
    You can explore the definition of an element in the standard libraries and see what predicates are available. Use the keyboard **F3** button to open the definition of any element. Alternatively, hover over the element and click **Jump to definition** in the tooltip displayed. The library file is opened in a new tab with the definition highlighted.
```

Comment:
Does this paragraph refer to LGTM or VSCode functionality? It seems a bit out of place now that neither is explicitly mentioned in this document. I would remove it, move it to a more appropriate document, or preface it with a clarification.

Looks good otherwise 👍

Reply:
Well spotted. This refers to LGTM functionality, so should probably be removed.

Reply:
Thanks for highlighting this. I'll check for it in other articles.